.github/workflows/check.yml

```diff
@@ -63,8 +63,15 @@ jobs:
                   components: rust-src, rustfmt, clippy
                   default: true
 
+            - name: Cache evcxr
+              id: cache-evcxr
+              uses: actions/cache@v2
+              with:
+                  path: ~/.cargo/bin/evcxr
```

Comment:
Probably we should cache whole `~/.cargo`, or, at least `~/.cargo/.crates.toml` and `~/.cargo/.crates2.json` - because we use `cargo install --list` command to determine whether evcxr is installed

Reply:
I tried to cache `~/.cargo` originally, but for some reason it got overwritten or it didn't work at all (the loaded cache was empty). But I didn't investigate further, I can try.